index.html

```diff
@@ -5,9 +5,639 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link href="./style.css" rel="stylesheet">
-    <title>NyTimes copy</title>
+    <script src="https://kit.fontawesome.com/d3a036121e.js"></script>
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun - The New York Times</title>
 </head>
 <body>
-    
+    <nav class="nav-bar">
+        <div class="nav-icon-container">
+            <img src="./resources/left-icons.png" alt="left side icons" class="nav-icon">
```

Comment:
You forget to put the img tag inside an anchor tag. The logo needs to be a link.